From cf3778631f87ad03274ac94dcae7c7fbbd036913 Mon Sep 17 00:00:00 2001
From: Kristian Fiskerstrand <kf@sumptuouscapital.com>
Date: Tue, 22 Nov 2016 10:07:33 +0100
Subject: [PATCH] gpgscm: Use shorter socket path lengts to improve test
 reliability

--
As socket lengths are normally restricted to 108 characters
(UNIX_PATH_MAX variable in /usr/include/linux/un.h), using 42 characters
by default easily results in errors.
---
 tests/gpgscm/tests.scm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/gpgscm/tests.scm b/tests/gpgscm/tests.scm
index 72afa99..039ea86 100644
--- a/tests/gpgscm/tests.scm
+++ b/tests/gpgscm/tests.scm
@@ -268,9 +268,9 @@
 (define (mkdtemp . components)
   (_mkdtemp (if (null? components)
 		(path-join (getenv "TMP")
-			   (string-append "gpgscm-" (get-isotime) "-"
+			   (string-append "gscm"
 					  (basename-suffix *scriptname* ".scm")
-					  "-XXXXXX"))
+					  "XXXXXX"))
 		(apply path-join components))))
 
 (macro (with-temporary-working-directory form)
-- 
2.7.3

